### ISSUE TITLE
Updating edge length after move

### ISSUE THREAD

**Issue by behrisch** [eclipse-sumo/sumo#11003]
When changing an edge geometry or moving the junctions and going directly to inspect mode the edge length is not updated (only after recalculating junction). the best solution would be to update at least the changed edge. If that is not possible a visual feedback showing that the length is not the current one would be appreciated.

**Comment by zapcity** [comment]
Recalculating edge length automatically would be much appreciated. Outside of that, how do we go about doing it manually?

**Comment by namdre** [comment]
press F5

**Comment by zapcity** [comment]
> press F5

F5 doesn't appear to do anything. What I do is:

1. Load my .net.xml file
2. `Set move mode`. I elongate one of the edges.
3. Press F5
4. `Set inspect mode`. I click the edge, and the `length` as listed under `Net: edge` hasn't changed since step 1. The `length` parameter should change in proportion to how much I change the visual length of the edge.

When I run my new network in a simulation, all the cars elongate and speed up, or shrink in length and slow down along that modified edge, according to whether I elongated/shrinked the actual edge.

**Comment by namdre** [comment]
in this case, your edge holds a custom length value. You can reset this to be re-computed by erasing the value from the length field.

**Comment by zapcity** [comment]
Thanks for that. It works. I noticed that the edge that wasn't updating it's length had it's length font colored black. Once I cleared it, it went to blue and was recalculated. Not sure why it was set to custom length in the first place. Appreciate your help.

**Comment by palvarezlopez** [comment]
This problem was already fixed in another ticket
